src/cephadm/cephadmlib/daemons/tracing.py

```diff
@@ -27,17 +22,17 @@ class Tracing(ContainerDaemonForm):
 
     components: Dict[str, Dict[str, Any]] = {
         'elasticsearch': {
-            'image': DEFAULT_ELASTICSEARCH_IMAGE,
+            'image': DefaultImages.ELASTICSEARCH.value.name,
             'envs': ['discovery.type=single-node'],
         },
         'jaeger-agent': {
-            'image': DEFAULT_JAEGER_AGENT_IMAGE,
+            'image': DefaultImages.JAEGER_AGENT.value.name,
         },
         'jaeger-collector': {
-            'image': DEFAULT_JAEGER_COLLECTOR_IMAGE,
+            'image': DefaultImages.JAEGER_COLLECTOR.value.name,
         },
         'jaeger-query': {
-            'image': DEFAULT_JAEGER_QUERY_IMAGE,
+            'image': DefaultImages.JAEGER_QUERY.value.name,
```

Comment:
to avoid this ".value.xxx" access we can add those properties to the `DefaultImages` class:

```
class DefaultImages(Enum):
    PROMETHEUS = ContainerImage(
        'quay.io/prometheus/prometheus:v2.51.0',
        'some_key_prometheus',
        'Prometheus container image'
    )

    @property
    def name(self):
        return self.value.name

    @property
    def key(self):
        return self.value.key

    @property
    def desc(self):
        return self.value.desc
```
This way, we can call `DefaultImages.JAEGER_COLLECTOR.name` directly

Reply:
Thanks for pointing this! I have done the changes.